Src/DependencyCollector/Shared/Implementation/FrameworkHttpEventListener.cs

```diff
@@ -78,13 +78,23 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
                 switch (eventData.EventId)
                 {
                     case BeginGetResponseEventId:
-                        this.OnBeginGetResponse(eventData);
+                        if (!DependencyTableStore.Instance.IsDesktopHttpDiagnosticSourceActivated)
+                        {
+                            // request is handled by Desktop DiagnosticSource Listener
+                            this.OnBeginGetResponse(eventData);
+                        }
+
                         break;
                     case EndGetResponseEventId:
                         this.OnEndGetResponse(eventData);
                         break;
                     case BeginGetRequestStreamEventId:
-                        this.OnBeginGetRequestStream(eventData);
+                        if (!DependencyTableStore.Instance.IsDesktopHttpDiagnosticSourceActivated)
+                        {
+                            // request is handled by Desktop DiagnosticSource Listener
+                            this.OnBeginGetRequestStream(eventData);
```

Comment:
In order to log something we need to get id from the payload, otherwise log will be useless. The whole point to avoid parsing payload. We will have log about callback from diagnostic source, it will be enough to understand that it was skipped by event source